Assets/__Scripts/Map/SpawnParameterHelper.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+
+public class SpawnParameterHelper : MonoBehaviour
```

Comment:
Considering this is all static methods, this class really does not need the extra baggage of `MonoBehaviour`. It's probably more fitting to have this be a `public static class`.

Reply:
makes sense